remoulade/state/backend.py

```diff
@@ -0,0 +1,62 @@
+from typing import Iterable
+
+from ..encoder import Encoder
+
+
+class State:
+    """Catalog Class, that
+    storages the state payload
+    """
+    def __init__(self, state):
+        self.state = state
+
+    def _asdict(self):
+        return {"state": self.state}
+
+
+Missing = type("Missing", (object,), {})()
```

Comment:
I don't think the Missing type is usefull, you can just return None. It is useful for the result backend as we needed to make a difference between result is None and no result.